### ISSUE TITLE
`_entrypoints.py`, `_scenario_1.py`, and `_scenario_2.py` are unused

### ISSUE THREAD

**Issue by garlic-os** [dbekaert/RAiDER#680]
**Describe the bug**
I cannot find any instance where a file imports from `_entrypoints.py`, `_scenario_1.py`, or _`scenario_2.py`, and pytest does not pick them up.

**To Reproduce**
1. Enumerate pytest tests and observe that the files in question are not present
2. Use vscode search/grep or similar to check for mentions of the filenames in the project and observe that they are never mentioned

**Expected behavior**
Are these tests intended to be used? If they are but are just out of commission right now perhaps we can add a comment to them to explain their situation for people looking at them.

**Desktop (please complete the following information):**
 - RAiDER git tag:
commit a9bf37cc1b82bc1837860a29a368ad22e6c7e96a (HEAD -> dev, tag: v0.5.3, upstream/main, upstream/dev, upstream/HEAD)
Merge: 2a34802 2d5eb95
Author: Charlie Marshak <cmarshak@users.noreply.github.com>
Date:   Tue Jul 30 11:17:05 2024 -0700
&nbsp;&nbsp;&nbsp;&nbsp;Merge pull request #678 from dbekaert/dev
&nbsp;&nbsp;&nbsp;&nbsp;v0.5.3
 - OS: Arch Linux 5.15.153.1-microsoft-standard-WSL2 on Windows 10

**Comment by jhkennedy** [comment]
@garlic-os I believe the intent with these tests was to temporarily disable them by removing the `test` prefix from the file name -- e.g., `test_entrypoints.py` was renamed to `_entrypoints.py` here:
https://github.com/dbekaert/RAiDER/pull/364#issuecomment-1297635852

`test_entrypoints.py` at least should be re-enabled to test that these CLI entry points all do exist and are installed properly:
https://github.com/dbekaert/RAiDER/blob/dev/pyproject.toml#L44-L51

**Comment by jlmaurer** [comment]
@garlic-os yes that's correct, these tests were temporarily disabled due as these were end-to-end tests that were downloading files at testing time instead of mocking and/or using cached files, with the result that they would hang if for example one of the weather model servers was out of commission. Some of these have been replaced in part by other tests (e.g., `test_synthetic`) but not fully. Eventually we'll need to re-visit and integrated them with the new tests that have been introduced.
